bsp4j/src/main/java/ch/epfl/scala/bsp4j/Protocol.xtend

```diff
@@ -436,6 +436,7 @@ class CompileReport {
   @NonNull Integer errors
   @NonNull Integer warnings
   Long time
+  Boolean noOp
```

Comment:
I wasn't 100% sure to make the `@NonNull` or not. I'm not a huge fan of having `Option[Boolean]` in bsp4s, but at the same time I know that some servers won't set this, and I want to possibly avoid having a server set this as `false` giving the false impression that it's not a no-op when maybe it actually is. Is this ok like this, or do you think it'd be better modeled as `@NonNull`?